msteams-platform/concepts/build-and-test/deep-links.md

```diff
@@ -205,9 +205,27 @@ microsoftTeams.executeDeepLink(/*deepLink*/);
 > [!NOTE]
 > In order to open the scheduling dialog in Teams, developers need to continue using the original deep-link URL based method, since Teams does not yet support the calendar capability.
 
-For more information about working with the calendar, see, the [calendar](/javascript/api/@microsoft/teams-js/calendar?view=msteams-client-js-latest&preserve-view=true) namespace in the API reference documentation.
+For more information about working with the calendar, see the [calendar](/javascript/api/@microsoft/teams-js/calendar?view=msteams-client-js-latest&preserve-view=true) namespace in the API reference documentation.
 
-### tab/Teams JS v1
+# [TeamsJS v2](#tab/teamsjs-v2)
+
+```javascript
```

Comment:
This is possibly a dumb question, but should this say typescript instead of javascript

Reply:
Fair question! :) Snippets across the Teams docs (and generally MS Docs) are marked JS, so I was following suit. Although TS is a superset of JS, I think somehow JS might present itself as the more general term (encompassing JS/TS scenarios)?